### ISSUE TITLE
Character display issue with syriac words in wordlist

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#404]
see issue #349.
I was able again to reproduce the character display issue in wordlist. Go to chrome and activate alpheios in Loeb classics.  double clcik on a couple of words then go to the syriac site sedra and change page language to syriac. double click on few words there. the words are saved in wordlist with character display issue see screenshot
![Screen Shot 2020-06-01 at 3 09 02 PM](https://user-images.githubusercontent.com/41396793/83412536-870b6f80-a41a-11ea-859c-16ddfcf927ec.png)

**Comment by monzug** [comment]
also in morphology pop-up
![Screen Shot 2020-06-01 at 3 16 03 PM](https://user-images.githubusercontent.com/41396793/83412762-f97c4f80-a41a-11ea-9844-e76072ceb93b.png)

and wordlist limit to this session
![Screen Shot 2020-06-01 at 3 16 38 PM](https://user-images.githubusercontent.com/41396793/83412796-06993e80-a41b-11ea-9ef6-0a7f4b4ac908.png)

**Comment by irina060981** [comment]
I was not able to reproduce - but I believe that this problem could be because of using not specific font
I have added lang attribute to wordlist to use a correct font

as for popup font problem - I believe we have some `ara` text there (I saw similiar for other words)

Finally I think that the problem coud be in the case that specific fonts was not loaded correctly.
We upload fonts from 
fonts.gstatic.com

May be there are some problems sometimes with uploading from there?
@balmas and @monzug , what do you think?

**Comment by monzug** [comment]
could be. I was working from my other location where the connection is not
too good.

**Comment by balmas** [comment]
>May be there are some problems sometimes with uploading from there?

Yes that seems possible.

**Comment by monzug** [comment]
let me try again

**Comment by monzug** [comment]
this is chrome on Mac:

![Screen Shot 2020-06-02 at 2 09 35 PM](https://user-images.githubusercontent.com/41396793/83518537-d9ac6080-a4da-11ea-8b85-02aa53a62bb5.png)

![Screen Shot 2020-06-02 at 2 10 03 PM](https://user-images.githubusercontent.com/41396793/83518530-d6b17000-a4da-11ea-9161-5c9c172805a0.png)

**Comment by balmas** [comment]
Chrome on Mac definitely has character encoding problems. Does the same problem occur on the page on which the words were looked up?

**Comment by monzug** [comment]
Chrome on PC - this is book Peshitta, acts of the apostles

![syriac-pc-chrome](https://user-images.githubusercontent.com/41396793/83519159-8241cc80-a4a9-11ea-94f8-d3d0f77bb68d.JPG)

**Comment by monzug** [comment]
Note that Loading is still happening in both environments

**Comment by balmas** [comment]
@monzug in that screenshot the characters look okay in moth Alpheios and on the underlying page?

**Comment by monzug** [comment]
lookup works fine in most of the words. just found one instance 

![Screen Shot 2020-06-02 at 2 19 55 PM](https://user-images.githubusercontent.com/41396793/83519463-53911980-a4dc-11ea-96ce-874c04d4c2c4.png)

**Comment by balmas** [comment]
Ok, so in the messages, that's probably because the font isn't being applied there.  That's a bug.

**Comment by monzug** [comment]
@balmas, sorry, I don't understand your question?

**Comment by irina060981** [comment]
I believe Bridget is talking about adding language attribute to the message (... was not found ...)
@balmas, it is insetered here from l10n messages and would need some refactoring - may be it is worth to create a separate issue for that?

**Comment by monzug** [comment]
Thanks Irina.

**Comment by balmas** [comment]
> I believe Bridget is talking about adding language attribute to the message (... was not found ...)
> @balmas, it is insetered here from l10n messages and would need some refactoring - may be it is worth to create a separate issue for that?

Yes, agree we need a new issue for that.

**Comment by balmas** [comment]
As for the display, here's what I see for that word ܗܘ in Chrome on Mac

![Screenshot 2020-06-02 08 29 24](https://user-images.githubusercontent.com/644870/83520460-b7e7b500-a4ab-11ea-8134-e7b762c85066.png)

What I was wondering is if Chrome was just having trouble displaying those characters in the definition for "the same", regardless of whether they were in Alpheios or on the underlying page. But I can't reproduce the problem in either place.

However, it's worth noting that that definition text, which comes from Sedra, does not properly identify the language of the characters, so we cannot apply the Syriac font to them. Why it is inconsistent behavior though I don't know.

**Comment by monzug** [comment]
Irina, can you enter that as you have a better understanding on what's going on. Thanks

**Comment by monzug** [comment]
ok, on Mac it seems that it takes longer to upload some fonts. Now that I have been playing with syriac for a while, the same word looks ok.

![Screen Shot 2020-06-02 at 2 39 31 PM](https://user-images.githubusercontent.com/41396793/83521185-2a25bd00-a4df-11ea-914d-bac2048ff171.png)

**Comment by monzug** [comment]
and an other one that worked before

![Screen Shot 2020-06-02 at 2 43 31 PM](https://user-images.githubusercontent.com/41396793/83521446-88eb3680-a4df-11ea-89ac-fe7829582a5a.png)

**Comment by balmas** [comment]
I think we can't do more than we are doing here. Chrome on Mac is notoriously bad at unicode fonts.

**Comment by monzug** [comment]
Like it's good on other thing...

**Comment by irina060981** [comment]
ok - fix with adding language to worditems is done

I will also refactor message component with lang attribute later

**Comment by irina060981** [comment]
ok I have created a separate issue for formatting message
Also I believe that we should define several fonts for arabic and syriac texts, this way we would have correct fallback 
@balmas , what do you think?

**Comment by balmas** [comment]
> ok I have created a separate issue for formatting message
> Also I believe that we should define several fonts for arabic and syriac texts, this way we would have correct fallback
> @balmas , what do you think?

Agree

**Comment by irina060981** [comment]
ok - fallback fonts are added
merged and closed

**Comment by monzug** [comment]
note for testing: lookup using one of those words with weird font works fine

**Comment by balmas** [comment]
Alpheios Reading Tools 3.3.0 build qa.20200605325
Alpheios Components 3.3.0-qa.20200605311

**Comment by monzug** [comment]
I was checking the log and got some error messages on fonts.gstatic.com in safari,  see screenshot
![Screen Shot 2020-06-08 at 5 04 08 PM](https://user-images.githubusercontent.com/41396793/84046343-42e41600-a9aa-11ea-9007-83776abf6e80.png)

I was not able to reproduce the character issue in wordlist. sometimes I see that the character is loading and then it displayed correctly, so the fix is verified.
@irina060981 and @balmas, do the errors in the log give you any interesting info?

**Comment by monzug** [comment]
I could not see those error messages in the log

**Comment by monzug** [comment]
I could not see those error messages in the log
